app/src/main/java/org/ole/planet/myplanet/base/BaseRecyclerFragment.java

```diff
@@ -267,4 +267,10 @@ private boolean isValidFilter(RealmMyLibrary l) {
         return (sub && lev && lan && med);
     }
 
+    protected void clearSelectedItems() {
+        for (int i = selectedItems.size() - 1; i >= 0; i--) {
```

Comment:
@mattscodeans  use selectedItems.clear()

Reply:
@rrijal53 I am pushing a new commit that fixes this now.